ISISNeutronMuon/MDANSE
# Add MDTraj converter and update to atom mapping code

**Description of work**
Added MDTraj converter and optimized the atom mapping code. 

Similarly to the MDAnalysis converter, the MDTraj converter converts trajectories with trajectory files and a topology file. Trajectory files can include one or more files but should only be of a single type e.g. xtc. In MDTraj topology files are optional if the trajectory files contain topology information.

**To test**
1) Convert some gromacs trajectories from the MDANSE-Examples repository. 
Enter PDB files into the topology file setting and trr or xtc files into the trajectory files setting and convert the trajectory. Atom mapping should be generated faster than before.
2) Convert some charmm trajectories from the MDANSE-Examples repository. 
Enter PDB files into the topology file setting and dcd files into the trajectory files setting and convert the trajectory.
3) Try converting a trajectory using multiple coordinate files.
Either generate multiple coordinate files or copy and rename them. Convert the trajectory, if you copy and renamed it then the MDANSE trajectory should be twice as long and repeat itself.
